### PR TITLE
Update pep-0008.txt

### PR DESCRIPTION
**Pull request by toonarmycaptain**
Clarified contradictions regarding comment formatting. NB Complete sentences should have a period.

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by toonarmycaptain** [comment]
Still waiting on registration confirmation for bug.s.python.org. Signed CLA and received confirmation.

**Comment by toonarmycaptain** [comment]
Do I need to close and resubmit with the appropriate form signed pre-pull request?

File: pep-0008.txt [hunk 1]
```diff
@@ -690,16 +690,17 @@ Comments that contradict the code are worse than no comments.  Always
 make a priority of keeping the comments up-to-date when the code
 changes!
 
-Comments should be complete sentences.  If a comment is a phrase or
-sentence, its first word should be capitalized, unless it is an
-identifier that begins with a lower case letter (never alter the case
-of identifiers!).
+Comments should be complete sentences (unless short).  If a comment is
```
Comment (pfmoore) [inline pep-0008.txt]: I disagree with this. I don't see any justification for allowing this.

(Or should I say "Disagree. No Justification."? ;-))

Comment (brettcannon) [reply pep-0008.txt]: I agree with Paul. I don't think there's ever a reasonable to reason to justify not writing complete sentences, no matter how short the comment is.

Comment (toonarmycaptain) [reply pep-0008.txt]: That means all comments should terminate with a period. Otherwise it is not a complete sentence.

Comment (brettcannon) [reply pep-0008.txt]: @toonarmycaptain yep.

Comment (toonarmycaptain) [reply pep-0008.txt]: I edited to remove reference to short comments not needing a period.

File: pep-0008.txt [hunk 2]
```diff
@@ -690,16 +690,15 @@ Comments that contradict the code are worse than no comments.  Always
...
 
-Comments should be complete sentences.  If a comment is a phrase or
-sentence, its first word should be capitalized, unless it is an
-identifier that begins with a lower case letter (never alter the case
-of identifiers!).
+Comments should be complete sentences.  The first word should be 
+capitalized, unless it is an identifier that begins with a lower case
+letter (never alter the case of identifiers!).
 
-If a comment is short, the period at the end can be omitted.  Block
```
Comment (toonarmycaptain) [inline pep-0008.txt]: Period never omitted because comments should be complete sentences.

File: pep-0008.txt [hunk 3]
```diff
@@ -690,16 +690,15 @@ Comments that contradict the code are worse than no comments.  Always
...
-comments generally consist of one or more paragraphs built out of
-complete sentences, and each sentence should end in a period.
+Block comments generally consist of one or more paragraphs built out of
+complete sentences, with each sentence ending in a period.
 
-You should use two spaces after a sentence-ending period.
+You should use two spaces after a sentence-ending period in multi
+sentence comments, except after the final sentence.
```
Comment (toonarmycaptain) [inline pep-0008.txt]: Explicitly ask for double space separated sentences, without suggesting trailing whitespace.

File: pep-0008.txt (hunk 1 above)
Comment (toonarmycaptain) [reply pep-0008.txt]: I agree @pfmoore. I only specified that to comply with the present iteration which states that "If a comment is short, the period at the end can be omitted."

**Comment by brettcannon** [comment]
@warsaw Current version work for you, or do you  want to close this PR?

**Comment by warsaw** [comment]
Thanks, this looks good to me now!

**Pull request merged**